manifest.toml

```diff
@@ -0,0 +1,210 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "gitea"
+name = "Gitea"
+description.en = "Lightweight Git forge"
+description.fr = "Forge Git légère"
+
+version = "1.20.4~ynh2"
+
+maintainers = ["Josué Tille"]
+# previous_maintainers = [
+#     "rafi59"
+# ]
+
+[upstream]
+license = "MIT"
+website = "https://gitea.io/"
+admindoc = "https://docs.gitea.io/"
+code = "https://github.com/go-gitea/gitea"
+cpe = "cpe:2.3:a:gitea:gitea"
+fund = "https://opencollective.com/gitea"
+
+[integration]
+yunohost = ">= 11.2"
+architectures = [
+    "amd64",
+    "i386",
+    "armhf",
+    "arm64"
+]
+multi_instance = true
+ldap = true
+sso = false
+disk = "120M"
+ram.build = "100M"
+ram.runtime = "100M"
+
+[install]
+    [install.domain]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "domain"
+
+    [install.path]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "path"
+    default = "/gitea"
+
+    [install.admin]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "A public server means that everybody is able to access to the main page of the forge, on the public profile of the user and on the public repository. But you still can limit the access to each repository if you set it as private. Note that to be able to use the remote Git command (clone, pull, push) with HTTP and to use the API by (by example with a smartphone), you need to set this application as public."
+    help.fr = "Un serveur public signifie que tout le monde peut accéder à la page principale de la forge, au profil public des utilisateurs et aux dépôts publics. Vous pouvez également définir les dépôts comme étant privés. Notez que pour pouvoir utiliser les commandes Git distantes (clone, pull, push) avec HTTP et pour pouvoir utiliser l'API (par exemple avec un smartphone), vous devez paramétrer cette application comme étant publique."
+    type = "group"
+    default = "visitors"
+
+[resources]
+
+    [resources.system_user]
+    # Hard-coded...
+    home = "/home/yunohost.app/__APP__"
+
+    [resources.install_dir]
+
+    [resources.data_dir]
+    dir = "/home/yunohost.app/__APP__"
```

Comment:
Can't we remove this? is it not the standard path?

Reply:
Oh yeah we can !